applications/nrf5340_audio/src/drivers/cs47l63_reg_conf.h

```diff
@@ -119,8 +121,14 @@ const uint32_t asp1_enable[][2] = {
 	{ CS47L63_GPIO4_CTRL1, 0xE1000000 },
 	{ CS47L63_GPIO5_CTRL1, 0x61000001 },
 
-	/* Set ASP1 in slave mode and 16 bit per channel */
+/* Set ASP1 in slave mode and 16 bit per channel */
+#if CONFIG_AUDIO_SAMPLE_RATE_16000_HZ
+	{ CS47L63_ASP1_CONTROL1, 0x021B },
+#elif CONFIG_AUDIO_SAMPLE_RATE_24000_HZ
+	{ CS47L63_ASP1_CONTROL1, 0x011B },
```

Comment:
Please check if we can use the defines from CS47L63 here instead

Reply:
This is how it is done in the rest of the file as well. Using defines for all settings will be a major rewrite, and not entirely sure it can be done. Had a look and you still need to say what bit to set and how many spots to shift it, so it doesn't add any value